### ISSUE TITLE
section-notice: aria-roledescription is on invalid element

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1461]
The `aria-roledescription` is invalid on a div with no role. It should be on the section tag instead.

![Screen Shot 2021-07-15 at 5 19 27 PM](https://user-images.githubusercontent.com/38065/125873530-42217ca5-22eb-49f6-a865-eb908169cd86.png)
